src/Common/tests/XunitTraitsDiscoverers/Category.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    internal struct XunitConstants
+    {
+        public const string NonWindowsTest = "nonwindowstests";
+        public const string NonLinuxTest = "nonlinuxtests";
+        public const string NonMacTest = "nonmactests";
```

Comment:
We've been using "OSX" instead of "Mac" elsewhere, such as in the project files and file names.  Can we do the same here?